.github/workflows/build-test-publish.yml

```diff
@@ -93,14 +85,6 @@ jobs:
       - name: Checkout
```

Comment:
This checkout step can be removed too (it was only needed so that this specific job could access/run the `bin/setup_docker_ci` script), plus the same checkout step in the `publish` jobs, plus also the `bin/setup_docker_ci` script itself.

Reply:
Fixed in https://github.com/heroku/cnb-builder-images/pull/529/commits/4064944309043bb95f5e33059fe46367269b526a.